src/preferences/dialog/dlgprefrhythm.cpp

```diff
@@ -0,0 +1,62 @@
+#include "preferences/dialog/dlgprefrhythm.h"
+
+#include "control/controlobject.h"
+#include "defs_urls.h"
+
+DlgPrefRhythm::DlgPrefRhythm(QWidget *parent, UserSettingsPointer pConfig)
+        : DlgPreferencePage(parent) {
+
+    setupUi(this);
+
+    loadSettings();
+
+    // TODO(Cristiano) Create remaining connections..  
+    connect(bBeatsAndTempoEnabled, SIGNAL(stateChanged(int)),
```

Comment:
These SIGNAL/SLOT macros are legacy from before Qt 5. Use the Qt 5 functor signal/slot syntax for new code.
https://wiki.qt.io/New_Signal_Slot_Syntax

Reply:
ping